backend/inspirehep/indexer/api.py

```diff
@@ -21,6 +21,7 @@ def get_references_to_update(record):
             uuids_to_reindex |= (
                 record.get_all_connected_records_uuids_of_modified_collaborations()
             )
+        uuids_to_reindex |= record.get_modified_journal_uuids()
```

Comment:
shouldn't we have it for experiments, institutions etc?

Reply:
actually we just need to update this one instead https://github.com/inspirehep/inspirehep/blob/master/backend/inspirehep/records/api/literature.py#L555-L575

Reply:
probably we should

Reply:
As we discussed let's move this into `record.get_linked_papers_if_reference_changed`